### PR TITLE
Making code more compliant to formatting standards (PEP...)

### PR DESCRIPTION
**Pull request by kenden**
Just running pycharm code inspector and pycharm reformat code

### CODE REVIEW

File: rtmbot.py [hunk 1]
```diff
@@ -38,48 +42,54 @@ def start(self):
...
         for plugin in self.bot_plugins:
             limiter = False
             for output in plugin.do_output():
                 channel = self.slack_client.server.channels.find(output[0])
-                if channel != None and output[1] != None:
-                    if limiter == True:
+                if channel is not None and output[1] is not None:
+                    if limiter:
                         time.sleep(.1)
-                        limiter = False
```
Comment (jammons) [inline rtmbot.py]: Why was this line removed? I think it changes the logic to remove it.

File: rtmbot.py [hunk 2]
```diff
@@ -38,48 +42,54 @@ def start(self):
...
-                    message = output[1].encode('ascii','ignore')
-                    channel.send_message("{}".format(message))
-                    limiter = True
+                    limiter = True  # TODO: check goal: no sleep for 1st channel, sleep of all after ?
+                    # TODO: find out how to safely encode stuff if needed :(
```
Comment (jammons) [inline rtmbot.py]: Can you resubmit the PR without changing these lines around the message sending?

File: rtmbot.py [hunk 3]
```diff
@@ -90,32 +100,36 @@ def __init__(self, name, plugin_config={}):
...
             self.module.crontable = []
+
     def do(self, function_name, data):
         if function_name in dir(self.module):
-            #this makes the plugin fail with stack trace in debug mode
+            # this makes the plugin fail with stack trace in debug mode
             if not debug:
                 try:
-                    eval("self.module."+function_name)(data)
+                    eval("self.module." + function_name)(data, bot, config)
```
Comment (jammons) [inline rtmbot.py]: Where did these come from?

**Comment by jammons** [comment]
Thanks for submitting this PR. I have a few questions, but would love to get the PEP 8 fixes merged if you can answer them / fix a couple things.

**Comment by jammons** [comment]
I made a few edits/added some code and merged this into master. Thanks for the help. Sorry it took so long.

**Comment by kenden** [comment]
Thanks!
